.github/workflows/release.yml

```diff
@@ -62,7 +52,7 @@ jobs:
   publish-s3:
     needs: create-packages
     runs-on: ubuntu-18.04
```

Comment:
Here and above we can move to `ubuntu-latest`: we have no specific set of dependencies of the `update_repo.sh` script anymore.

Reply:
Migrated in new commit